config/nginx.conf.template

```diff
@@ -231,6 +231,12 @@ http {
       proxy_pass http://portainer:9000/;
     }
 
+    location /studio/ {
```

Comment:
Let's not do this one yet, since we don't have a way to secure it (e.g., our instance is wide open to anyone).  Only change the development template.